spec/models/comment_spec.rb

```diff
@@ -0,0 +1,26 @@
+require 'rails_helper'
+
+RSpec.describe Comment, type: :model do
+  subject do
+    Comment.create do |comment|
+      comment.post = Post.create do |post|
+        post.title = 'Post 1'
```

Comment:
- [ ] The same here. So far great job 👏🏻 However,  you are missing a test to test the `update_comments_counter` method of your Comment model, please add this ✍🏻

```ruby
  def update_comments_counter
    post.increment!(:comments_counter)
  end
```